.github/workflows/_hugo.yml

```
@@ -0,0 +1,55 @@
+            --baseURL "https://jetthoughts.com/" \
+            --environment production
+
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v3
+        with:
+          path: ./public
+
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ inputs.PAGE_URL }}
```

actionlint

property "page_url" is not defined in object type {}

(expression)